storage/jellyfish-merkle/src/jellyfish_merkle_test.rs

```diff
@@ -637,6 +637,44 @@ proptest! {
         test_existent_keys_impl(&tree, version, &kvs);
     }
 
+    #[test]
+    fn test_put_blob_with_proof2(
```

Comment:
why calling it xxx2? (because there seems to be no xxx1)

Reply:
Because it tests 2 accounts

Reply:
I don't feel this test is a lot different from the one above except that it adds a few assertions. Maybe just add these assertions to `init_mock_db` so every test gets them?